packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -120,6 +121,7 @@ const AccountSwitcher = ({
         closeAccountsDialog();
         if (account_loginid === loginid) return;
         await switchAccount(loginid);
+        RudderStack.setAccountType(loginid.substring(0, 2));
```

Comment:
<img width="506" alt="Screenshot 2023-07-18 at 3 10 34 PM" src="https://github.com/binary-com/deriv-app/assets/118344354/e51fcdc2-f8b5-4967-8479-1e3528ef5e89">
Since authorize gets called again everytime account is switched anyways.

Reply:
Yeah, in this case it's only called when account switched, which is what I want to achieve. If we call it from authorize, it would be executed whenever the call is made, which might trigger additional events when we don't want it.